.decent_ci-Windows.yaml

```diff
@@ -2,5 +2,5 @@ compilers:
   - name: Visual Studio
     version: 16
     architecture: Win64
-    cmake_extra_flags: -DBUILD_FORTRAN:BOOL=ON -DBUILD_TESTING:BOOL=ON -DCOMMIT_SHA=%COMMIT_SHA% -DENABLE_GTEST_DEBUG_MODE:BOOL=OFF -DLINK_WITH_PYTHON=ON -DPYTHON_EXECUTABLE:PATH=C:/Users/elee/AppData/Local/Programs/Python/Python311/python.exe -DPython_EXECUTABLE:PATH=C:/Users/elee/AppData/Local/Programs/Python/Python311/python.exe
+    cmake_extra_flags: -DBUILD_FORTRAN:BOOL=ON -DBUILD_TESTING:BOOL=ON -DCOMMIT_SHA=%COMMIT_SHA% -DENABLE_GTEST_DEBUG_MODE:BOOL=OFF -DLINK_WITH_PYTHON=ON -DPython_EXECUTABLE:PATH=C:/Users/elee/AppData/Local/Programs/Python/Python311/python.exe
```

Comment:
Thanks to @jmarrec for confirming this cleanup, the PYTHON version isn't needed, so it's removed, and Decent CI is still very happy on Windows.